src/Components/CustomElements/src/js/package.json

```diff
@@ -5,25 +5,18 @@
     "clean": "rimraf ./dist/Debug ./dist/Release",
     "prebuild": "npm run clean",
     "build": "npm run build:debug && npm run build:production",
-    "build:debug": "webpack --mode development --config ./webpack.config.js",
-    "build:production": "webpack --mode production --config ./webpack.config.js"
+    "build:debug": "rollup -c --environment development",
+    "build:production": "rollup -c --environment production"
   },
   "devDependencies": {
-    "@babel/core": "^7.15.0",
-    "@babel/preset-env": "^7.15.0",
-    "@typescript-eslint/eslint-plugin": "^5.26.0",
-    "@typescript-eslint/parser": "^5.26.0",
-    "eslint": "^8.16.0",
-    "inspectpack": "^4.7.1",
+    "@rollup/plugin-commonjs": "^25.0.7",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@rollup/plugin-replace": "^5.0.5",
+    "@rollup/plugin-terser": "^0.4.4",
+    "@rollup/plugin-typescript": "^11.1.5",
```

Comment:
Are all of these needed? I'm surprised by `plugin-commonjs` since it doesn't seem like we're using that format.